content/realtime/channel-params.textile

```diff
@@ -139,3 +139,111 @@ h3(#rewind-examples-mqtt). Rewind example with MQTT
     ...
   });
 ```
+h2(#delta). Delta parameter
+
+The @delta@ channel parameter expresses the intent to receive the delta (diff) between the previous and current message instead of the current message in full on a given channel. This parameter is oblivious to the sender and only affects the subscriber using it. I.e. the sender sends messages in full and Ably Realtime would compute and send the message deltas to any subscriber using this parameter. 
+
+It should also be noted that Ably Realtime does not guarantee every message to be a delta once the @delta@ parameter is specified. Ably Realtime might chose to send some messages in full depending on system load, etc. So a subscriber using the @delta@ parameter should be prepared to handle a random mix of delta and full messages.
```

Comment:
> It should also be noted that Ably Realtime does not guarantee every message to be a delta once the `delta` parameter is specified

the infinitive ("...guarantee every message to be a delta"), while not wrong, is a relatively unusual, stilted construction (especially as most of our documentation is written with a fairly casual tone). Suggested rewording: "Note that requesting deltas does not guarantee that every message will be a delta."

Reply:
fbf72fa